papaya-web/src/main/java/pl/edu/agh/papaya/mappers/SprintMapper.java

```diff
@@ -0,0 +1,43 @@
+package pl.edu.agh.papaya.mappers;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.edu.agh.papaya.api.model.SprintDto;
+import pl.edu.agh.papaya.model.Sprint;
+
+@Component
+public class SprintMapper extends Mapper<Sprint, SprintDto> {
+
+    @Autowired
+    private LocalDateTimePeriodMapper localDateTimePeriodMapper;
+
+    @Autowired
+    private SprintStateMapper sprintStateMapper;
+
+    public List<SprintDto> mapToApi(List<Sprint> modelItems, LocalDateTime evaluationTime) {
+        return modelItems.stream()
+                .map(x -> mapToApi(x, evaluationTime))
```

Comment:
ej, ej
ale nazwy znaczące

Reply:
Haha, nie zauważyłam

Reply:
No pewnie

Reply:
Zawsze najpierw robię x, sprawdzam, czy IntelliJ mi nie powie, że można uprościć i jak nie, to zmieniam na znaczącą

Reply:
jeszcze kiedyś Cię przyłapię na... nie chce mi to przejść przez gardło... BRAKU ENTERA NA KOŃCU PLIKU

Reply:
Aaaa, ALE JUŻ JEST - ja byłam w środku intensywnego refactoringu akurat

Reply:
A czekaj, był czy nie było?

Reply:
Aaaa, był